docs/modules/deployment/pages/core/system-requirements.adoc

```diff
@@ -2,51 +2,89 @@
 [[system-requirements-core]]
 = Minimum System Requirements
 
-[options="header, autowidth"]
+The following hardware specs are the minimum required to run {page-component-title}:
+
+[options="autowidth"]
 |===
-|                        | Just testing*           | Minimum server specification**
-| *CPU*                  | 2 GHz dual core x86_64  | 3 GHz quad core x86_64 and above
-| *RAM*                  | 4 GB (physical)         | 16 GB (physical) and above
-| *Storage (disk space)* | 50 GB HDD, SSD          | 1 TB with SSD and above
+|   | Just Testing^1^   | Minimum Server Specification^2^
+
+| *CPU*
+| 2GHz dual core x86_64
+| 3GHz quad core x86_64 and above
+
+| *RAM*
+| 4GB (physical)
+| 16GB (physical) and above
+
+| *Storage (disk space)*
+| 50-GB HDD, SSD
+| 1TB with SSD and above
 |===
 
-*You can install the packages and the services start. +
-**Your intended workload (network size, number of monitored metrics, flows, events, and data retention requirements) affects these requirements.
+^1^ You can install the packages, and the {page-component-title} services start. +
+^2^ Your intended workload (network size, number of monitored metrics, flows, events, and data retention requirements) affects these requirements.
 
 [[operating-systems-core]]
 == Operating systems
 
-OpenNMS {page-component-title} requires one of the following operating systems.
-See the release notes for information about platform support changes, including deprecated platforms.
+OpenNMS {page-component-title} requires one of the following operating systems:
 
-[options="header, autowidth"]
+[options="autowidth"]
 |===
-| Operating System        | Compatible Versions (64-bit)
-| RHEL                    | {compatible-rhel7} +
-                            {compatible-rhel8}
-| CentOS                  | {compatible-centos7} +
-                            {compatible-centos-stream}
+| Operating System  | Compatible Versions (64-bit)
+
+| RHEL
+| {compatible-rhel7} +
+{compatible-rhel8}
+
+| CentOS
+| {compatible-centos7} +
+{compatible-centos-stream}
+
 ifeval::["{page-component-title}" == "Horizon"]
-| Debian                  | {compatible-debian}
-| Ubuntu                  | {compatible-ubuntu}
+| Debian
+| {compatible-debian}
+
+| Ubuntu
+| {compatible-ubuntu}
 endif::[]
 |===
 
+NOTE: See the xref:releasenotes:whatsnew.adoc[release notes] for information about platform support changes, including deprecated platforms.
+
 [[system-components-core]]
 == Compatibility
 
 {page-component-title} {full-display-version} is compatible with the following components:
 
-[options="header, autowidth"]
+[options="autowidth"]
```

Comment:
Might be nice to indicate which of these are required (Java, Postgres, RRDTool, Browser) and which are optional (Helm, Plugin, Cassandra, Elastic, Kafka)